aries_cloudagent/anoncreds/default/legacy_indy/registry.py

```diff
@@ -636,15 +636,23 @@ async def register_revocation_list(
             profile, rev_list, rev_reg_def.type, rev_reg_entry
         )
 
+        seq_no = None
+        try:
+            seq_no = rev_entry_res["result"]["txnMetadata"]["seqNo"]
```

Comment:
Similar question, under what circumstances do we not see the seqNo in the return?

Reply:
This one can happen when you try and publish a rev reg def that was already posted to the ledger. It would get a 500 KeyError because txnMetadata was missing. If the result is returned without the sequence number then it gets a 400 error with a duplicate entry response now.